etc/osm-smartrak.osm2pgsql.mml

```diff
@@ -475,7 +475,7 @@
         "key_field": "", 
         "project": "foss4g-2011", 
         "srs": "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0.0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs +over", 
-        "table": "( SELECT way, highway AS type, name, ref, oneway, CHAR_LENGTH(ref) AS reflen\n    FROM planet_osm_line\n    WHERE highway IN ('motorway', 'trunk')\n      AND (name IS NOT NULL OR ref IS NOT NULL)\n) AS data", 
+        "table": "(SELECT way, type, name, ref, oneway, reflen, array_length(refs,1) AS height, (SELECT MAX(char_length(ref)) FROM unnest(refs) AS u(ref)) AS width, array_to_string(refs, E'\n') AS refs FROM (SELECT way, highway AS type, name, ref, oneway, CHAR_LENGTH(ref) AS reflen, string_to_array(ref, ';') AS refs FROM planet_osm_line WHERE highway IN ('motorway', 'trunk')  AND (name IS NOT NULL OR ref IS NOT NULL)) AS intermediate_data) AS data", 
```

Comment:
This is merges the query from the openstreetmap-carto `osm2pgsql` [file](https://github.com/gravitystorm/openstreetmap-carto/blob/master/project.mml#L1802) with the existing query so that the old `ref` and `reflen` fields are still accessible in the default style files.

I considered making a separate `osm2pgsql` file altogether for skate, but I ran into some trouble with how the `osm2pgsql` file [that is used](https://github.com/jacobtoye/osm-bright/blob/dd3cfcd6e1fd942133577553c7fae44468f406e8/make.py#L107) is defined by `config.py`, which is used when the container is built rather than when it is run like the existing `STYLE_DIR` environment variable.